Closes #24: Add code to publish to plugins.gradle.org.

mcomella opened the pull request:
Once approved, the plugin will be added to:
  https://plugins.gradle.org/plugin/org.mozilla.apt

I used a mozilla-specific account to publish:
  https://plugins.gradle.org/u/mcomella-moz

mcomella committed:
Closes #24: Add code to publish to plugins.gradle.org.

Once approved, the plugin will be added to:
  https://plugins.gradle.org/plugin/org.mozilla.apt

I used a mozilla-specific account to publish:
  https://plugins.gradle.org/u/mcomella-moz

pocmo reviewed (approved):
👍

pocmo commented on gradle-plugin/build.gradle.kts:
```diff
@@ -25,3 +26,20 @@ dependencies {
+        create("mozillaPlugin") { // this identifier is unused on remote
+            id = "org.mozilla.apt"
+            displayName = "Mozilla APT Plugin"
+            description = "A plugin for the Android Product Team (APT) at Mozilla"
```
Minor nit: Not sure whether I would use "apt" here. This plugin could become useful for other teams at Mozilla building Android apps (AC, Lockbox, Taipei, Test Pilot). Locking back even the name "APT" may not stay that way forever. :)

pocmo replied:
Minor nit: Not sure whether I would use "apt" here. This plugin could become useful for other teams at Mozilla building Android apps (AC, Lockbox, Taipei, Test Pilot). Locking back even the name "APT" may not stay that way forever. :)

pocmo commented on gradle-plugin/build.gradle.kts:
```diff
@@ -25,3 +26,20 @@ dependencies {
+            id = "org.mozilla.apt"
+            displayName = "Mozilla APT Plugin"
+            description = "A plugin for the Android Product Team (APT) at Mozilla"
+            implementationClass = "org.mozilla.apt.MozillaPlugin"
```
Minor nit: I am not sure if the name should include "APT". This plugin looks like it could be useful to other teams at Mozilla building Android apps (e.g. AC, Lockbox, Taipei, Test Pilot, ..). And looking back I'm not even sure if the "APT" team will use that name forever. :)

pocmo commented on gradle-plugin/build.gradle.kts:
```diff
@@ -25,3 +26,20 @@ dependencies {
+            id = "org.mozilla.apt"
+            displayName = "Mozilla APT Plugin"
+            description = "A plugin for the Android Product Team (APT) at Mozilla"
+            implementationClass = "org.mozilla.apt.MozillaPlugin"
```
Minor nit: I am not sure if the name should include "APT". This plugin looks like it could be useful to other teams at Mozilla building Android apps (e.g. AC, Lockbox, Taipei, Test Pilot, ..). And looking back I'm not even sure if the "APT" team will use that name forever. :)

mcomella replied:
@pocmo Do you have an alternative suggestion? Maybe `org.mozilla.android.MozillaPlugin`?

pocmo replied:
I don't. But your suggestion sounds good to me. :)

pocmo commented:
Argh, GitHub is slow an unreliably today. That has caused my comment to get published a thousand times. https://status.github.com/messages

mcomella committed:
Issue #24 - review: Change apt -> android in package.

This plugin can be used by Android teams, not just APT. Also, the APT
might change their name at some point.

mcomella commented:
The plugin will be published at https://plugins.gradle.org/plugin/org.mozilla.android
